doc/parameters/param_list.md

```diff
@@ -429,15 +429,15 @@ When `controller_plugins`\`progress_checker_plugin`\`goal_checker_plugin` parame
 | ----------| --------| ------------|
 | node_names | N/A | Ordered list of node names to bringup through lifecycle transition |
 | autostart | false | Whether to transition nodes to active state on startup |
-| bond_timeout_ms | 4000 | Timeout for bond to fail if no heartbeat can be found, in milliseconds. If set to 0, it will be disabled. Must be larger than 300ms for stable bringup. |
+| bond_timeout | 4 | Timeout for bond to fail if no heartbeat can be found, in milliseconds. If set to 0, it will be disabled. Must be larger than 0.3s for stable bringup. |
```

Comment:
This should be a double now, so 4.0 seconds, so that people can still represent milliseconds. Also, the description needs to be updated (no longer ms)